modules/stochastic_tools/test/tests/surrogates/libtorch_nn/retrain.i

```diff
@@ -0,0 +1,64 @@
+[StochasticTools]
+[]
+
+[Samplers]
+  [sample]
```

Comment:
We change this to "train"  or "train_sample" to indicate this sample is used for training

Reply:
Changed!